httpsign/random.go

```diff
@@ -74,6 +74,7 @@ type SeededRNG struct {
 // generated by a math/rand.Rand.
 func (r *SeededRNG) bytes(bytes int) ([]byte, error) {
 	if r.rand == nil {
+		//nolint: gosec // TODO: use crypto/math random number generator.
```

Comment:
There is the next comment a few lines above:

> Not cryptographically secure, not thread safe.

Reply:
Revised comment.